src/daft-sql/src/modules/mod.rs

```diff
@@ -21,12 +19,10 @@ pub mod window;
 
 pub use aggs::SQLModuleAggs;
 pub use config::SQLModuleConfig;
-pub use float::SQLModuleFloat;
 pub use image::SQLModuleImage;
 pub use json::SQLModuleJson;
 pub use list::SQLModuleList;
 pub use map::SQLModuleMap;
-pub use numeric::SQLModuleNumeric;
```

Comment:
Do we plan on migrating the rest of these namespaces other than float and numeric as well? What is the work required for that?

Reply:
yes this is in progress in a few subsequent PR's

- [images](https://github.com/Eventual-Inc/Daft/pull/4294)
- [utf8](https://github.com/Eventual-Inc/Daft/pull/4312)
- [json](https://github.com/Eventual-Inc/Daft/pull/4302)

I plan on moving them all over in the coming weeks.